extract-project-assets

```diff
@@ -0,0 +1,4 @@
+echo "copying project.assets.json from container"
+cp -r ./obj/FFCDemoPaymentService/project.assets.json ./FFCDemoPaymentService/obj/project.assets.json
+echo "making project.assets.json writable by all so Jenkins can delete it"
+chmod 666 ./FFCDemoPaymentService/obj/project.assets.json
```

Comment:
The other repos have a `scripts` folder, would it make sense to add one to this repo, or as we only have one script is it not worth it?

Reply:
Thanks, I think it's worth being consistent, I'll move the file.